app/javascript/components/ReviewTile.js

```diff
@@ -16,9 +17,17 @@ const ReviewTile = props => {
         <div id="review_comment">
           {Comment} {props.commentbody}
         </div>
-        <div>
+        <div className="editReviewLink">
```

Comment:
Class names should be conventially created with dashes instead of camelCase: `edit-review-link`: and be sure to update CSS too!